data/schema/pbl.cfg

```diff
@@ -34,9 +33,25 @@
 				{SIMPLE_KEY core string}
 			[/then]
 		[/if]
+		[if]
+			[not]
+				forum_auth=yes
+			[/not]
+			[then]
+				{SIMPLE_KEY passphrase string}
+				{SIMPLE_KEY email email}
+			[/then]
+		[/if]
 		[tag]
 			name="feedback"
```

Comment:
Although there's a `feedback` tag identified here, when passed the current version of Non-Fatal Wounds, https://github.com/stevecotton/Non_Fatal_Wounds/blob/d03b4e3/_server.pbl , it gives the error
`Tag [feedback] may not be used in []`.

Reply:
Looks like I messed up the last-second change to make it use `get_wml_location`. Pushing a fix asap.